core/src/components/item/item.ionic.scss

```diff
@@ -0,0 +1,119 @@
+
+// Item
+// --------------------------------------------------
+
+:host {
+  /**
+    * @prop --background: Background of the item
+    * @prop --background-activated: Background of the item when pressed. Note: setting this will interfere with the Material Design ripple.
+    * @prop --background-activated-opacity: Opacity of the item background when pressed
+    * @prop --background-focused: Background of the item when focused with the tab key
+    * @prop --background-focused-opacity: Opacity of the item background when focused with the tab key
+    * @prop --background-hover: Background of the item on hover
+    * @prop --background-hover-opacity: Opacity of the background of the item on hover
+    *
+    * @prop --border-color: Color of the item border
+    * @prop --border-radius: Radius of the item border
+    * @prop --border-style: Style of the item border
+    * @prop --border-width: Width of the item border
+    *
+    * @prop --color: Color of the item
+    * @prop --color-activated: Color of the item when pressed
+    * @prop --color-focused: Color of the item when focused with the tab key
+    * @prop --color-hover: Color of the item on hover
+    *
+    * @prop --detail-icon-color: Color of the item detail icon
+    * @prop --detail-icon-opacity: Opacity of the item detail icon
+    * @prop --detail-icon-font-size: Font size of the item detail icon
+    *
+    * @prop --inner-border-width: Width of the item inner border
+    * @prop --inner-box-shadow: Box shadow of the item inner
+    * @prop --inner-padding-top: Top padding of the item inner
+    * @prop --inner-padding-end: Right padding if direction is left-to-right, and left padding if direction is right-to-left of the item inner
+    * @prop --inner-padding-bottom: Bottom padding of the item inner
+    * @prop --inner-padding-start: Left padding if direction is left-to-right, and right padding if direction is right-to-left of the item inner
+    *
+    * @prop --min-height: Minimum height of the item
+    *
+    * @prop --padding-bottom: Bottom padding of the item
+    * @prop --padding-end: Right padding if direction is left-to-right, and left padding if direction is right-to-left of the item
+    * @prop --padding-start: Left padding if direction is left-to-right, and right padding if direction is right-to-left of the item
+    * @prop --padding-top: Top padding of the item
+    *
+    * @prop --transition: Transition of the item
+    *
+    * @prop --ripple-color: Color of the item ripple effect
+    *
+    */
+  --border-color: #{globals.$ionic-color-neutral-200};
+  --border-style: #{globals.$ionic-border-style-solid};
+  --border-width: #{globals.$ionic-border-size-025};
+  --default-background: #{globals.$ionic-color-base-white};
+  --item-min-height: #{globals.$ionic-scale-1800};
+  --item-width: #{globals.$ionic-line-height-full};
+  --space-200: #{globals.$ionic-space-200};
+  --space-400: #{globals.$ionic-space-400};
```

Comment:
Why create css variables for these two space values? We should use the tokens directly where needed in this case I think.

Unless, you think this is important to have available as a public api to be easily customized. But if that's the case, we should come up with a different name.

Reply:
I suggested changes to use `--padding` variables if we want to match how we have allowed users to customize the padding in the past.

Reply:
@brandyscarney that sounds good to me yeah 👍